podcasts/TranscriptLoader.swift

```diff
@@ -0,0 +1,87 @@
+import Foundation
+import SwiftSubtitles
+import PocketCastsDataModel
+
+class TranscriptLoader {
```

Comment:
Since this is loading and parsing, maybe we could call it `TranscriptCoordinator`?